public/doc/openapi-3-v0.0.2.json

```diff
@@ -1825,6 +1828,9 @@
                 },
                 {
                   "$ref": "#/components/schemas/InventoryCollectionVmSecurityGroup"
+                },
+                {
+                  "$ref": "#/components/schemas/InventoryCollectionServiceInstanceTask"
```

Comment:
It appears this is not needed for the `refresh_type` change.  Was this added for targeted refresh?

Reply:
no it was added in PR https://github.com/RedHatInsights/topological_inventory-core/pull/187 and autogenerated as a new inventory collection. So we probably didn't run openapi:generate between PR 187 and this one